packages/customer-account-ui-extensions-react/src/components/Card/Card.ts

```diff
@@ -0,0 +1,9 @@
+import {Card as BaseCard} from '@shopify/customer-account-ui-extensions';
```

Comment:
we have already switched to the unstable version, do we still want to update it main?

Reply:
cc @lihao ?

Reply:
Cool then I think for now it's fine to update both `main` and `unstable`